packages/meld.rb

```diff
@@ -1,33 +1,31 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Meld < Package
+class Meld < Meson
   description 'Meld is a visual diff and merge tool targeted at developers.'
   homepage 'https://meldmerge.org/'
-  version '3.22.0'
+  version '3.22.0-684e1e2-py3.12'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'https://gitlab.gnome.org/GNOME/meld/-/archive/3.22.0/meld-3.22.0.tar.gz'
-  source_sha256 '6332dda01925a2ee367b4b2c50da9a89e040b6656f6643c4aae037c20690a8a2'
+  source_url 'https://gitlab.gnome.org/GNOME/meld.git'
+  git_hashtag '684e1e27f31b4f0053da546a273a4ecf07df595f'
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '41addb40a83c9994b929eb2279f4a2ba1a9daca533ffb716a7e0e78010397061',
-     armv7l: '41addb40a83c9994b929eb2279f4a2ba1a9daca533ffb716a7e0e78010397061',
+    aarch64: 'c9044c2ee4f70158c0f271a764f475c6a7cb095525699fe20c3ab6e55c3a622d',
+     armv7l: 'c9044c2ee4f70158c0f271a764f475c6a7cb095525699fe20c3ab6e55c3a622d',
        i686: '738c8fe2c1df886e38587f69dddaecb928d615e19233ccc5c4fdb83f8c552478',
```

Comment:
Since you changed the version, unless you reuploaded the i686 binary under that new version, the package will try and download a non-existent i686 binary and fail.

Reply:
In that case, does it make sense to also remove the i686 filelist? (i'm not actually sure, so genuinely asking)

Reply:
Yes!

Reply:
It turns out that there is no manifest file for i686 for meld...
```
 ls manifest/i686/m/me
mediainfo.filelist   megatools.filelist   memcached.filelist   mesonbuild.filelist
```